plotly/offline/offline.py

```diff
@@ -56,6 +56,74 @@ def init_notebook_mode():
                  '</script>'))
 
 
+def plot(figure_or_data, html_style='standalone', show_link=True, link_text='Export to plot.ly', validate=True):
+    """ Configured to work with localhost Plotly graph viewer
+    """
+    EMBED_TYPES = ('standalone', 'body', 'div')
+    if html_style not in EMBED_TYPES:
+        raise ValueError('html_style must be one of %s' % ', '.join(EMBED_TYPES))
+
+    html = ""
+    if html_style == 'standalone':
+        html += '<html><head><script type="text/javascript">' + get_plotlyjs() + '</script></head><body>'
```

Comment:
we should include
`<!DOCTYPE html>` and  `<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">`